From 3bbd8cd5c2882e0d69a456006c14320434a6b198 Mon Sep 17 00:00:00 2001
From: Ivan Vecera <ivecera@redhat.com>
Date: Mon, 8 Sep 2014 16:13:36 -0400
Subject: [ethernet] be2net: log LPVID used in multi-channel configs

Message-id: <1410192881-30702-14-git-send-email-ivecera@redhat.com>
Patchwork-id: 92415
O-Subject: [RHEL7.1 PATCH 13/78] be2net: log LPVID used in multi-channel configs
Bugzilla: 1088575
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>

BZ: #1088575

Upstream commit(s):
commit bdac85b55e57ca880147a4c6bd9f8af69507956a
Author: Ravikumar Nelavelli <ravikumar.nelavelli@emulex.com>
Date:   Tue Mar 11 18:53:05 2014 +0530

    be2net: log LPVID used in multi-channel configs

    Signed-off-by: Ravikumar Nelavelli <ravikumar.nelavelli@emulex.com>
    Signed-off-by: Sathya Perla <sathya.perla@emulex.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Ivan Vecera <ivecera@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/emulex/benet/be_cmds.c b/drivers/net/ethernet/emulex/benet/be_cmds.c
index ff353d7..cf5afe7 100644
--- a/drivers/net/ethernet/emulex/benet/be_cmds.c
+++ b/drivers/net/ethernet/emulex/benet/be_cmds.c
@@ -244,10 +244,12 @@ static void be_async_grp5_qos_speed_process(struct be_adapter *adapter,
 static void be_async_grp5_pvid_state_process(struct be_adapter *adapter,
   struct be_async_event_grp5_pvid_state *evt)
 {
- if (evt->enabled)
+ if (evt->enabled) {
   adapter->pvid = le16_to_cpu(evt->tag) & VLAN_VID_MASK;
- else
+  dev_info(&adapter->pdev->dev, "LPVID: %d\n", adapter->pvid);
+ } else {
   adapter->pvid = 0;
+ }
 }
 
 static void be_async_grp5_evt_process(struct be_adapter *adapter,
-- 
1.7.1